canonical/ubuntu.com
# WD-2345 - Add banner for GA real time kernel

## Done

- Add banner for real-time kernel

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- https://ubuntu-com-12712.demos.haus/internet-of-things
- https://ubuntu-com-12712.demos.haus/telco
- https://ubuntu-com-12712.demos.haus/internet-of-things
- https://ubuntu-com-12712.demos.haus/core
- https://ubuntu-com-12712.demos.haus/embedded

## Issue / Card

Fixes WD-2345
